1-exercises/B-todo-list/script.js

```diff
@@ -1,7 +1,24 @@
+
+const contentDiv = document.querySelector("#content")
+
 function todoList(todos) {
-  // Write your code here...
-}
 
+  let list = document.createElement("ol");
+  contentDiv.appendChild(list);
+
+  todos.forEach(arrayElement => {
+     let listItem = document.createElement("li");
+     list.appendChild(listItem);
+
+     listItem.innerText = arrayElement.todo;
+
+     listItem.addEventListener("click", () => {
+     listItem.style.textDecoration === "line-through" ?  listItem.style.textDecoration = "":
```

Comment:
Perfect! Why don't you use the class you defined in the CSS file? instead of adding the style here? :D